homeassistant/components/sensor/teamspeak.py

```diff
@@ -0,0 +1,128 @@
+"""
+homeassistant.components.sensor.teamspeak
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Monitors a teamspeak server
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.teamspeak/
+"""
+import logging
+from homeassistant.helpers.entity import Entity
+
+REQUIREMENTS = ['ts3==0.7.1']
+_LOGGER = logging.getLogger(__name__)
+
+DEFAULT_NAME = 'Teamspeak Server'
+ATTR_SERVER_NAME = 'Name'
+ATTR_MAX_USERS = 'Max. Users'
```

Comment:
It's up to you if you want to add the attributes to the sensor. Not sure if something like max users is really useful.

Attributes are allowed to have spaces and periods in them but it is bad practice. For example, printing an attribute in a template has the format `state.attributes.max_users`. As you can see, that wouldn't work with periods or spaces in the attribute name.